lib/dndtable.sty

```diff
@@ -9,11 +9,31 @@
 \newcolumntype{Y}{>{\centering\arraybackslash}X}
 
 % Table Environment
-\newenvironment{dndtable}[1][XX]{
-	\par\vspace*{8pt}
-	\noindent
-	\fontfamily{lmss}\selectfont %Select font
-	\rowcolors{1}{}{commentgreen} % Alternate colors
-	\tabularx{\linewidth}{#1}
-	}
-	{\endtabularx\vspace{8pt plus 1pt}\noindent} 
+
+% \tabularx expands to commands that directly hack the typesetting system. The
+% macro expects and end token, similar to a real environment.
+% Wrapping an environment that uses tabularx causes a runaway argument error
+% because tabularx cannot find the end token.
+% Instead, create two command that expand to the table header and footer.
+\newcommand{\dnd@starttable}[1]{%
```

Comment:
I do not think we need this, but I am not completely against it. dnditemtable is deprecated by this change, and I cannot think of other reasons to nest the dndtable environment (doesn't mean there aren't any). This is part of the tradeoff of the easier column layout of tabularx I guess.